src/main/webapp/populateData.js

```diff
@@ -90,12 +90,24 @@ function populateGmail() {
  * Populate Tasks container with user information
  */
 function populateTasks() {
-  // Get Container for Tasks content
-  const tasksContainer = document.querySelector('#tasks');
+  let fetchFrom;
+  const select = document.querySelector('#tasks-select');
+  // Cast from HTMLOptionsCollection to Array
+  const options = Array(...select.options);
 
-  // Get list of tasks from user's Tasks account
-  // and display the task titles from all task lists on the screen
-  fetch('/tasks')
+  if (options.length === 0) {
+    fetchFrom = '/tasks';
+  } else {
+    const selectedOptions = [];
+    options.forEach((option) => {
+      if (option.selected) {
+        selectedOptions.push(option.value);
+      }
+    });
+    fetchFrom = '/tasks?selected=' + selectedOptions.join();
```

Comment:
What does "join()" use as the default delimiter? Do you need to escape the delimiter if it is found in the task list names?

(You want to be consistent in what delimiter you use across the entire codebase.)

Reply:
Good point, `join()` uses `,` however that seems to be a valid character for task list names - this seems to be the case for every key on the keyboard and I don't see why they would make any of the characters invalid either. Will bring this up on the standup today!

Reply:
From our standup today - we agreed to map task list names to task list IDs and use the task list ID as a means of identifying to exact task list to summarize key information from.